src/test/scala/uk/gov/hmrc/test/ui/pages/SharedActions.scala

```diff
@@ -17,14 +17,41 @@
 package uk.gov.hmrc.test.ui.pages
 
 import org.openqa.selenium.By
-import uk.gov.hmrc.test.ui.pages.registration.RegistrationLoginPage.authoritySubmitButton
 
 object SharedActions extends BasePage {
 
+  val authoritySubmitButton = By.id("submit-top")
+  def clickLinkByPartialText(value: String): Unit ={
+    click(By.partialLinkText(value))
+  }
+
   def authorityWizardSubmitButton(): Unit =
     clickElement(authoritySubmitButton)
 
   def assertPartialTextIsDisplayed(value: String): Unit =
     assert(getText(By.id("main-content")).contains(value))
 
+  def clickById(id: String): Unit = {
+    driver.findElement(By.id(id)).click()
+  }
+
+  def assertPage(value: String) = {
+    assert(getText(By.id("main-content")).contains(value))
+  }
+
+  def selectLabelByPartialText(value: String): Unit = {
+    if (value == "") {
+      driver.findElement(By.cssSelector(".govuk-button")).click()
+      assertPage("Select your UK turnover for {0}")
+  }
+    else {
+    click(By.xpath(s"//label[contains(text(), '$value')]"))
+    driver.findElement(By.cssSelector(".govuk-button")).click()
+    value match {
+        case "Less than £10.2 million" => assertPage("You do not need to register for the Economic Crime Levy")
+        case "Equal to or more than £10.2 million" => assertPage("Who is your Anti-Money Laundering (AML) supervisor?")
+    }
+    }
```

Comment:
Not quite sure why you need this 

The radio buttons have IDs assigned to them:

`less-than` and `equal-or-more-than`, so can you not use these to select ?

Reply:
I have used if else logic, any suggestion?